bootstrap.example.toml

```diff
@@ -106,6 +106,9 @@
 # Whether to build LLVM with support for it's gpu offload runtime.
 #llvm.offload = false
 
+# absolute path to ClangConfig.cmake
```

Comment:
It's a bit weird that it is called `dir`, but the comment states that it should be a path to a file. Should it be a path to a directory that contains this file?

Reply:
yes, it's the dir in which the file is.

Reply:
Ok, then please update the comment :)